jekyll/_cci2/security-recommendations.md

```diff
@@ -107,4 +107,4 @@ EOF
 {: #see-also }
 {:.no_toc}
 
-[GitHub and Bitbucket Integration]({{ site.baseurl }}/gh-bb-integration/)
+[GitHub and Bitbucket Integration]({{ site.baseurl }}/github-integration/)
```

Comment:
@rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

Reply:
sounds good!